quickstarts/dotnet/ado-dotnet/config.yml

```diff
@@ -0,0 +1,53 @@
+
+slug: ado-dotnet
+description: |
+  ## Comprehensive monitoring quickstart for ADO.net
+
+  When you monitor ADO.net with New Relic, you can observe your ADO.NET data set in real time. Our .NET agent will also let you correlate transactions moving across your application environment and troubleshoot while working in a live development environment.
+
+
+  ## Why monitor ADO.NET?
+
+  Discovering data access API calls can help in identifying the following issues:
+    - Inaccessible databases or issues with the network library.
+    - Database and client software schema incompatibility.
+    - Problems occur when different ADO.NET components interact with one another or your own components.
+    - Whether hard-coded or created by an application, incorrect SQL.
+    - Flawed programming logic.
+
+  ## What’s included in this quickstart?
+
+  The ADO.NET monitoring quickstart from New Relic provides immediate full-stack observability:
+    - Alerts (high load connection, hard connects exceeding max limit, pool size exceeding optimum limit).
+    - Dashboards (hard connects per second, hard disconnects per second, number of active connections, soft connects per second,  and more).
```

Comment:
Observed extra space after "second,"

I feel, text need to be like below:
Dashboards (hard connects per second, hard disconnects per second, number of active connections, soft connects per second and more).